<launch>
  <node name="icp_node" type="icp_ros_node" pkg="icp_ros" ns="r1">
    <param name="base_frame"         value="r1/base_link"/>
    <param name="base_laser_frame"  value="r1/base_scan"/>
    <param name="odom_frame"        value="r1/odom"/>
    <param name="map_frame"         value="map"/>

    <param name="update_time"             value="0.0"/>
    <param name="use_sim_time"            value="false"/>
    <param name="time_threshold"          value="1"/>
    <param name="angle_upper_threshold"   value="1.57"/>
    <param name="dist_upper_threshold"    value="1.0"/>
    <param name="angle_threshold"         value="0.01"/>
    <param name="update_age_threshold"    value="1"/>
    <param name="dist_threshold"          value="0.01"/>
    <param name="icp_inlier_threshold"    value="0.88"/>
    <param name="icp_inlier_dist"         value="0.3"/>
    <param name="icp_num_iter"            value="100"/>
    <param name="pose_covariance_trans"   value="0.5"/>
    <param name="scan_rate"               value="25"/>
  </node>
</launch>
